src/clover/simulation/clinic.py

```diff
@@ -0,0 +1,335 @@
+#!/usr/bin/python3
+########################################################################################
+# clinic.py - Health-centre clinic module                                              #
+#                                                                                      #
+# Author: Ilaria Del Frate                                                             #
+# Copyright: Ilaria Del Frate, 2022                                                    #
+# License: Open source                                                                 #
+# Most recent update: 26/07/2022                                                       #
+#                                                                                      #
+# For more information, please email:                                                  #
+#     philip.sandwell@gmail.com                                                        #
+########################################################################################
+
+"""
+clinic.py - Works out the health centre load.
+
+"""
+
+import dataclasses
+
+from typing import List
+
+import yaml
+
+import numpy as np
+import pandas as pd
+import matplotlib.pyplot as plt
+
+
+@dataclasses.dataclass
+class Clinic:
+    """
+    Represents a clinic.
+
+    .. attribute:: name
+        The name of the clinic.
+
+    """
+
+    name: str
+
+    # Transmission Load
+    floor_area: float  # Tells python that a clinic has an area, and it's a number
+    roof_area: float
+    surface_area_walls: float
+    surface_area_doors_windows: float
+    u_value_walls: float
+    u_value_windows_doors: float
+    u_value_floor: float
+    u_value_roof: float
+    inside_ideal_temperature: float
+    # outside_temperature: float
+
+    # Internal heat load - people
+    people_1: float
+    people_2: float
+    heat_loss_people_1: float
+    heat_loss_people_2: float
+    start_time_people_1: List[int]
+    end_time_people_1: List[int]
+    start_time_people_2: List[int]
+    end_time_people_2: List[int]
+
+    # Internal heat load - Lighting
+    lamps_internal: float
+    lamps_external: float
+    start_time_lamps: List[int]
+    end_time_lamps: List[int]
+    time_lamps_external: float
+    wattage_lamps: float
+
+    # Equipment heat load
+    fridge_wattage: float
+    time_fridge: float
+
+    # Infiltration loads
+    changes: float
+    volume_cold_store: float
+    energy_new_air: float
+    start_time_infiltration: List[int]
+    end_time_infiltration: List[int]
+
+    run_hours: float
+
+
+MY_PATH = "C:/Users/Ilaria/CLOVER/locations/Bahraich/inputs/simulation/"
+with open(MY_PATH + "clinic.yaml") as f:
+    data_clinic = yaml.load(f, Loader=yaml.FullLoader)
+#  print(data_clinic)
+
+
+MY_CLINIC: Clinic = Clinic(**data_clinic["clinics"][0])
```

Comment:
### :bug: Hardcoding
We wrote the `_parse_clinic_inputs` function for the purpose of parsing the information into clinics. Here, you're effectively doing the work that that function should be doing, and that it will need to be doing in order to run as part of the wider CLOVER code :smile: